### PR TITLE
Cut through handcuffs with power wirecutters

### PR DESCRIPTION
**Pull request by psykzz**
:cl: PsyKzz
add: Jaws of life can now cut handcuffs
/:cl:

[why]: # Just seems logical to be able to.

### CODE REVIEW

**Comment by psykzz** [comment]
This shouldn't be merged until #33821 gets merged (if it does at all).

File: code/game/objects/items/tools/wirecutters.dm [hunk 1]
```diff
@@ -92,4 +95,16 @@
...
 	to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
 	qdel(src)
-	user.put_in_active_hand(pryjaws)
\ No newline at end of file
+	user.put_in_active_hand(pryjaws)
+	
+/obj/item/wirecutters/power/attack(mob/living/carbon/C, mob/user)
+	if(istype(C) && C.handcuffed)
+		user.visible_message("<span class='notice'>[user] cuts [C]'s restraints with [src]!</span>")
+		qdel(C.handcuffed)
```
Comment (ShizCalev) [inline code/game/objects/items/tools/wirecutters.dm]: When #33821 is merged, simplify this to just the qdel

Comment (psykzz) [reply code/game/objects/items/tools/wirecutters.dm]: Will do it now, as I wanted to wait until your PR was done anyway

**Review by ShizCalev — APPROVED** [review]

**Comment by jughu** [comment]
I disagree as power jaws are incredible common and can be mass produced making securities day much harder and probaly tempt them go lethal even more

What you could do instead is make eswords and such cut em

**Comment by PaznerIV** [comment]
Not as common as you think as they need mats to produced and engineers tend to forget about the protolath in there department, also most engineers I see don't tend to hand out power tools like candy to the crew.

**Comment by jughu** [comment]
No but im afraid this possibility changes the meta because wirecutters cut trough zipties instantly mainlly the reason officers use handcuffs over cableties/zipties

**Comment by kevinz000** [comment]
this is bad but not as bad as when science could mass produce them now engi has to do it.

**Comment by Bawhoppen** [comment]
Probably fine actually

**Comment by psykzz** [comment]
Always up for reverting it if it becomes a problem. thought it would give some more depth to the tools.

**Comment by Arzion5** [comment]
What about the heavy-duty wirecutters that traitors can have and alien wirecutters?

**Pull request merged**